Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\r\n# as it contains information specific to your local configuration.\r\n#\r\n# Location of the SDK. This is only used by Gradle.\r\n# For customization when using a Version Control System, please read the\r\n# header note.\r\n#Wed Jun 30 21:17:01 EET 2021\r\nsdk.dir=C\\:\\\\Users\\\\Almansy\\\\AppData\\\\Local\\\\Android\\\\Sdk\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision 8b2cb6796d1f91a5a432aeff1ac631e001c3221b)
+++ b/local.properties	(date 1654323051659)
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Wed Jun 30 21:17:01 EET 2021
-sdk.dir=C\:\\Users\\Almansy\\AppData\\Local\\Android\\Sdk
+#Sat Jun 04 08:10:51 EET 2022
+sdk.dir=C\:\\Users\\Arwa\\AppData\\Local\\Android\\Sdk
